ae/core/playwright_manager.py

```diff
@@ -391,6 +422,16 @@ def log_system_message(self, message: str):
         """
         self.ui_manager.new_system_message(message)
 
+    async def update_processing_state(self, processing_state: str):
+        """
+        Update the processing state of the overlay.
+
+        Args:
+            is_processing (str): "init", "processing", "done"
```

Comment:
this would be best put in an enum. But we can fix that at some point, maybe just a `#TODO:` as reminder to address this?